bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1556,6 +1560,15 @@ lnd/clm2/surfdata_map/release-clm5.0.30/surfdata_ne0np4.CONUS.ne30x8_hist_78pfts
 <lai_mapalgo  hgrid="1x1_asphaltjungleNJ" >nn</lai_mapalgo>
 <lai_mapalgo  hgrid="5x5_amazon"          >nn</lai_mapalgo>
 
+<!-- crop calendar streams namelist defaults -->
+<stream_year_first_cropcal  >1850</stream_year_first_cropcal>
+<stream_year_last_cropcal   >2100</stream_year_last_cropcal>
+<model_year_align_cropcal   >1850</model_year_align_cropcal>
+<stream_meshfile_cropcal    hgrid="360x720cru"       >/glade/p/cesmdata/cseg/inputdata/share/meshes/10x15_nomask_c110308_ESMFmesh.nc</stream_meshfile_cropcal>
+
+<stream_fldfilename_sdate hgrid="360x720cru"       >/glade/u/home/samrabin/crop_dates/sdates_ggcmi_crop_calendar_phase3_v1.01_nninterp-f10_f10_mg37.2000-2000.nc</stream_fldfilename_sdate>
+<stream_fldfilename_cultivar_gdds hgrid="360x720cru">/glade/u/home/samrabin/crop_dates/gdds_20220121_155415.nc</stream_fldfilename_cultivar_gdds>
```

Comment:
Yeah, I'm not happy with the defaults situation. I wanted to set defaults at half-degree resolution, but I never ended up generating the sowing date file—or even trickier, the cultivar GDD file, since that'd require a half-degree run. I think the best thing to do for now would be to just not have any defaults. Is that an option @ekluzek ?

Reply:
> I think the best thing to do for now would be to just not have any defaults. Is that an option?

I know you directed this to Erik, but I'll give my opinion: I feel that's totally reasonable for something like this that is not yet expected to completely work out-of-the-box.

Reply:
Removed default files in c2e4c676e.